webpack/assets/javascripts/react_app/redux/API/APIHelpers.js

```diff
@@ -0,0 +1,19 @@
+import { API_OPERATIONS } from './APIConstants';
+import { API } from './';
+
+export const getApiMethodByActionType = type => type.substring(4).toLowerCase();
+
+export const getApiResponse = async ({ type, url, headers, params }) => {
+  const method = getApiMethodByActionType(type);
+  switch (method) {
+    case 'get':
+      return API[method](url, headers, params);
+    case 'delete':
+      return API[method](url, headers);
+    default:
+      return API[method](url, params, headers);
```

Comment:
Would be nice as a follow-up PR to refactor `redux/API/API.js` so that all its methods take arguments in the same order.  Or even better, have the methods take a single object that can be destructured, like this one does.

Reply:
yeah, I would even like to use `axios` directly from here